caluma/caluma_form/tests/__snapshots__/test_question.ambr

```diff
@@ -245,6 +245,7 @@
             '__typename': 'FloatQuestion',
             'floatMaxValue': 1.0,
             'floatMinValue': 0.0,
+            'floatStep': None,
```

Comment:
Can we add a test case that includes a value?

Reply:
Done